lib/nylas/restful_model.rb

```diff
@@ -61,8 +61,6 @@ def as_json(options = {})
     end
 
     def update(http_method, action, data = {}, params = {})
-      http_method = http_method.downcase
```

Comment:
Hmm, we sure we don't need this?

Reply:
@zspencer before the recent changes it was needed to pass a correct method name to `send`. now we pass `http_method` as a `RestClient` parameter so it does all the stuff with downcasing.

Reply:
👍 awesome! That makes sense to me!